http/action/src/main/java/io/knotx/databridge/http/action/HttpActionOptions.java

```diff
@@ -31,6 +31,7 @@
   private EndpointOptions endpointOptions;
   private ResponseOptions responseOptions;
   private long requestTimeoutMs;
+  private String actionLogLevel;
```

Comment:
This field is serialised - it should be named `logLevel`.

Reply:
fixed